tianshou/highlevel/config.py

```diff
@@ -67,7 +70,7 @@ class SamplingConfig(ToStringMixin):
     """the total size of the sample/replay buffer, in which environment steps (transitions) are
     stored"""
 
-    step_per_collect: int = 2048
+    step_per_collect: int | None = 2048
```

Comment:
Docstring must state that this is mutually exclusive with `episode_per_collect` and that one of them must be set.

Reply:
Done